{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -26,20 +26,26 @@ authors = [
 classifiers = [ # ToDo: Modify according to your needs!
     "Development Status :: 2 - Pre-Alpha",
     "Programming Language :: Python",
+    "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Programming Language :: Python :: Implementation :: CPython",
 ]
 # direct dependencies of this package, installed when users `pip install {{ cookiecutter.project_slug }}` later.
 dependencies = [ # ToDo: Modify according to your needs!
-    "typer",
+    "setuptools_scm",
```

Comment:
setuptools_scm should not be needed for a normal, i.e. non-development, installation. It's not necessary to have it as a runtime requirement.

Reply:
I agree but it is the issue as above: it is the hatch-vcs-footgun in __init__.py ...

Reply:
Argh, I did miss that I eliminated the setuptools_scm dependency already because of using hatch compared to the original footgun-version. the setuptools_scm did not work correctly with the version bumping.

Reply:
BTW - the issue with setuptools_scm was discussed here 
https://github.com/maresb/hatch-vcs-footgun-example/issues/3